packages/components/codemods/utils/getImportAlias.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { parseJsx } from "../__tests__"
+import { getImportAlias } from "./getImportAlias"
+
+describe("getImportAlias", () => {
+  it("returns the import name if it matches the target specifier", () => {
+    const input = parseJsx(`
+      import { Well } from "@kaizen/components"
+      export const TestComponent = () => <div><Well>Test</Well></div>`)
```

Comment:
Can remove the usage since it's unnecessary for the function :)

Reply:
Don't quite follow, feels like there's three cases that we need to test for this util?

Reply:
Gotcha now, have removed the redundant code code from the test :)